src/applications/static-pages/events/components/Results/index.js

```diff
@@ -149,6 +149,18 @@ export const Results = ({
                     </p>
 
                     <div className="vads-u-display--flex vads-u-flex-direction--column">
+                      {event.fieldFacilityLocation?.entity?.entityUrl.path && (
+                        <p className="vads-u-margin--0">
+                          <a
+                            href={
+                              event.fieldFacilityLocation?.entity?.entityUrl
+                                ?.path
+                            }
+                          >
+                            {event.fieldFacilityLocation?.entity?.title}
```

Comment:
What happens if title is not populated?

Reply:
I would rather be safe than sorry, I'll adjust the first conditional to this:

```
{event.fieldFacilityLocation?.entity?.entityUrl?.path &&
    event.fieldFacilityLocation?.entity?.title &&
```